### PR TITLE
Added command overrides to ECS Slack notification messages.

### PR DESCRIPTION
**Pull request by nick-alloy**

### CODE REVIEW

**Comment by alex** [comment]
linter

File: lambdas/ecs-events-to-slack/notify.py [hunk 1]
```diff
@@ -62,6 +68,11 @@ def message_formatter(event, region):
             message_color = "danger"
 
     attachment_fields = [
+        {
+            "title": "Command",
+            "value": command_formatted,
+            "short": "true"
+        },
```
Comment (alex) [inline lambdas/ecs-events-to-slack/notify.py:75]: From a UX perspective, what do you think about simply not including this field if it's the default?

Comment (nick-alloy) [reply lambdas/ecs-events-to-slack/notify.py:75]: I'm fine with that.  Update incoming.

Comment (alex) [reply lambdas/ecs-events-to-slack/notify.py:75]: As written you're including the value 2x :-)

Comment (nick-alloy) [reply lambdas/ecs-events-to-slack/notify.py]: 🤦

**Pull request merged**